lblod/app-lokaal-mandatenbeheer
# Ben/lmb 576 add verkiezingsdata new bestuursorgaan

## Description

This PR removes migrations that added installatievergaderingen for the subset of bestuurseenheden that consisted of our data before we got the data from loket. 
Also this PR adds a temporary migration that moves verkiezingsdata from the previous bestuursorgaan in de tijd to the new bestuursorgaan in de tijd. This way the preparation of the legislature has testdata to prepare the new legislature.

## How to test

You should remove your data/db unfortunately for this (except toLoad). And rerun your migrations. After your migrations have ran, you should see all gemeente bestuurseenheden now have an installatievergadering with relevant verkiezingsdata. Just randomly pick a few bestuurseenheden and see if this is indeed correct.

## Links to other PR's

- https://github.com/lblod/app-lokaal-mandatenbeheer/pull/168
This PR builds on the PR above, best to just run the current PR and review both this linked PR as this one.
